lib/rails/generators/channel/templates/channel.coffee.erb

```diff
@@ -0,0 +1,3 @@
+App.<%= channel_name %> = App.cable.subscriptions.create "<%= channel_class %>",
```

Comment:
We need to declare `@App = {}` somewhere for this to work. But I guess that can come with the integration with Rails as well, where we can add that to application.js.